src/components/DatePicker/RangePicker.jsx

```diff
@@ -97,16 +122,50 @@ export const RangePicker = ({
   const handleCanChange = pos => value =>
     currentMonth.clone().add(pos, 'month').isSame(moment.utc(value), 'month')
 
+  const computeSecondInterval = ({ start, end }, compareMethod) => {
+    const _start = moment(start)
+    const _end = moment(end)
+
+    const diff = _end.diff(_start, 'days') + 1
+
+    return {
+      start: _start.subtract(diff, 'days').format(COMMON_FORMAT),
+      end: _end.subtract(diff, 'days').format(COMMON_FORMAT)
+    }
+  }
```

Comment:
Do you need the `compareMethod` parameter inside this method?

Reply:
Yes, for other methods (e.g. last year): https://github.com/etvascom/etvas-internals/pull/41/commits/de3a047bf558cbf23da8bc1b11f9cf288aaf7ae2